db/data/20211110161259_queue_updates_for_trainees_with_mismapped_subjects.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class QueueUpdatesForTraineesWithMismappedSubjects < ActiveRecord::Migration[6.1]
+  def up
+    affected_subjects = %w[
+      "applied chemistry"
```

Comment:
Couple of issues: 
1. I think this syntax isn't working quite right when I ran this definition:
`=> ["\"applied", "chemistry\"", "biology", "\"business", "studies\"", `
```suggestion
      applied\ chemistry
```
2. Some of the subjects are stored in non-lowercase, eg: "Latin language" so it might be better to use `LOWER` in the query as well

Reply:
good catch 🤌